site/src/sphinx/advanced-structured-logging.rst

```diff
@@ -148,6 +152,75 @@ automatically for you:
         }
     }
 
+Enable previewing the contents
+------------------------------
+Armeria provides the ``requestContentPreview`` and ``responseContentPreview`` properties in :api:`RequestLog`
+to retrieve the content.
```

Comment:
How about: .. to retrieve the textual representation of the first N bytes of the request and response content.